core/templates/dev/head/domain/collection/CollectionNodeObjectFactory.js

```diff
@@ -97,8 +115,7 @@ oppia.factory('CollectionNodeObjectFactory', [
         acquired_skills: [],
         prerequisite_skills: [],
         exploration: {
-          exists: true,
-          newlyCreated: true
+          exists: true
```

Comment:
That's because there is no title/status/etc. I removed the object entirely, since that's indicative of a non-existent exploration (which is the acceptable default value).